runtime/tests/mock.rs

```diff
@@ -82,13 +100,48 @@ impl ExtBuilder {
 				ARDA_ASSET_ID,
 			],
 			initial_balance: self.initial_balance,
-			endowed_accounts: vec![alice(), bob(), charlie(), dave(), eve(), ferdie()],
+			endowed_accounts: vec![
+				alice(),
+				bob(),
+				charlie(),
+				dave(),
+				eve(),
+				ferdie(),
+				validators()[0].0.clone(),
+				validators()[1].0.clone(),
+				validators()[2].0.clone(),
+			],
 			next_asset_id: NEXT_ASSET_ID,
 			staking_asset_id: STAKING_ASSET_ID,
 			spending_asset_id: SPENDING_ASSET_ID,
 		}
 		.assimilate_storage(&mut t)
 		.unwrap();
+
+		let stash = self.initial_balance / 5;
```

Comment:
Instead of this pattern, we could add `stash` field in `struct ExtBuilder` and a setter method